app/assets/stylesheets/application.css

```diff
@@ -42,180 +43,198 @@
-  color: #E45A84;
+  color: #e45a84;
   top: 83%;
   left: 62%;
   animation-duration: 28s;
   animation-delay: -23s;
   transform-origin: -20vw -17vh;
   box-shadow: -58vmin 0 7.60177307072966vmin currentColor;
 }
+
 .background span:nth-child(8) {
-  color: #583C87;
+  color: #583c87;
   top: 5%;
   left: 45%;
   animation-duration: 71s;
   animation-delay: -73s;
   transform-origin: 18vw 2vh;
   box-shadow: -58vmin 0 7.849767183235642vmin currentColor;
 }
+
 .background span:nth-child(9) {
-  color: #583C87;
+  color: #583c87;
   top: 69%;
   left: 61%;
   animation-duration: 23s;
   animation-delay: -33s;
   transform-origin: -7vw -9vh;
   box-shadow: 58vmin 0 7.76240129368311vmin currentColor;
 }
+
 .background span:nth-child(10) {
-  color: #583C87;
+  color: #583c87;
   top: 71%;
   left: 73%;
   animation-duration: 39s;
   animation-delay: -68s;
   transform-origin: -23vw -20vh;
   box-shadow: 58vmin 0 7.5319761452181275vmin currentColor;
 }
+
 .background span:nth-child(11) {
-  color: #583C87;
+  color: #583c87;
   top: 13%;
   left: 21%;
   animation-duration: 30s;
   animation-delay: -92s;
   transform-origin: -21vw -18vh;
   box-shadow: 58vmin 0 8.061869498655287vmin currentColor;
 }
+
 .background span:nth-child(12) {
-  color: #583C87;
+  color: #583c87;
   top: 57%;
   left: 92%;
   animation-duration: 108s;
   animation-delay: -109s;
   transform-origin: -13vw -24vh;
   box-shadow: -58vmin 0 7.388159408341347vmin currentColor;
 }
+
 .background span:nth-child(13) {
-  color: #583C87;
+  color: #583c87;
   top: 46%;
   left: 27%;
   animation-duration: 48s;
   animation-delay: -69s;
   transform-origin: 22vw 20vh;
   box-shadow: 58vmin 0 7.636544399461244vmin currentColor;
 }
+
 .background span:nth-child(14) {
-  color: #FFACAC;
+  color: #ffacac;
   top: 94%;
   left: 21%;
   animation-duration: 105s;
   animation-delay: -18s;
   transform-origin: -24vw 11vh;
   box-shadow: -58vmin 0 7.376342725251474vmin currentColor;
 }
+
 .background span:nth-child(15) {
-  color: #FFACAC;
+  color: #ffacac;
   top: 88%;
   left: 97%;
   animation-duration: 8s;
   animation-delay: -24s;
   transform-origin: -15vw 7vh;
   box-shadow: -58vmin 0 7.421007705649089vmin currentColor;
 }
+
 .background span:nth-child(16) {
-  color: #E45A84;
+  color: #e45a84;
   top: 49%;
   left: 30%;
   animation-duration: 38s;
   animation-delay: -2s;
   transform-origin: -6vw -9vh;
   box-shadow: -58vmin 0 7.889817306272075vmin currentColor;
 }
+
 .background span:nth-child(17) {
-  color: #583C87;
+  color: #583c87;
   top: 59%;
   left: 25%;
   animation-duration: 110s;
   animation-delay: -70s;
   transform-origin: 17vw 9vh;
   box-shadow: 58vmin 0 8.099672871179418vmin currentColor;
 }
+
 .background span:nth-child(18) {
-  color: #E45A84;
+  color: #e45a84;
   top: 94%;
   left: 88%;
   animation-duration: 57s;
   animation-delay: -100s;
   transform-origin: -22vw 7vh;
   box-shadow: -58vmin 0 7.9246528170907vmin currentColor;
 }
```

Comment:
[ ] 3 tests are failing, kindly work on them to pass successfully.

- ![image](https://user-images.githubusercontent.com/55635977/164732632-46763ed7-e2ad-4aa6-a818-420c634de728.png)

[ ] Kindly, make sure that text `< class="d-grid gap-2 col-6 mx-auto">` won't appear before save button. 

- ![image](https://user-images.githubusercontent.com/55635977/164732819-14419606-2028-4583-b813-955660c89c1d.png)

Reply:
@Emmyn5600 Thank you for your review. I have pushed the required changes